packaging/standalone/standalone-enterprise/src/main/distribution/text/enterprise/conf/neo4j.conf

```diff
@@ -137,6 +137,18 @@ dbms.connector.https.enabled=true
 # Maximum number of history files for the query log.
 #dbms.logs.query.rotation.keep_number=7
 
+# Log parameters for executed queries (this is enabled by default).
+#dbms.logs.query.parameter_logging_enabled=true
+
+# Uncomment this line to include detailed time information for executed queries:
+#dbms.logs.query.time_logging_enabled=true
+
+# Uncomment this line to include bytes allocated by executed queries:
+#dbms.logs.query.allocation_logging_enabled=true
+
+# Uncomment this line to include page hits and page faults information for executed queries:
+#dbms.logs.query.page_logging_enabled=true
+
```

Comment:
Hmm, reading the documentation vs this seems slightly off

* `dbms.logs.query.time_logging_enabled`: the docs mention `queries that took longer than the configured threshold` whereas the comment indicates that details regarding all queries will be logged

I think we should add this at the top of this changeset so it is clear what threshold we are talking about and to make sure people enable the global setting:

```
# Uncomment this line to enable query logging
#dbms.logs.query.enabled=true

# Log only queries exceeding this threshold
#dbms.logs.query.threshold=0s
```